changelog/unreleased/add-file-type-filter-chip.md

```diff
@@ -0,0 +1,88 @@
+Enhancement: Add search mimetype filter
+
+Add filter MimeType filter shortcuts to search for specific document types.
+For example, a search query MimeType:documents will search for files with the following mimetypes:
```

Comment:
I find it rather confusing, that the mime type collection filter also needs `MimeType:` as keyword... can we choose another keyword for the collections? Like e.g. `FileType:` for the collection filters and `MimeType:` for the real mime type filters?

Reply:
The `MimeType` is key in bleve search. We can replace the `FileType:` with `MimeType:` or add it as an alias.
But it will bring some mess because the users usually associate a `FileType` with the file extension, and users could expect that `FileType:*docx` returns the a.docx, b.docx, etc.

Reply:
Luckily users don't write the query themselves but use a dedicated ui filter component. :-) So the problem you're describing is only developer facing.

Do you have another idea for a semantically clear key for `something something collection of mimetypes`?

Reply:
The webteam just proposes `fileCategory`. I'm ok with it. Only one question. Have it be an alias for `MimeType` or the new key that supports only keywords ` file, folder, document
spreadsheet
presentation
pdf
image
video
audio
archive` ?

Reply:
> Luckily users don't write the query themselves but use a dedicated ui filter component. :-) So the problem you're describing is only developer facing.
> 
Users can write a search query

Reply:
@kulmann @lookacat  Also we can use `contenttype` because in response this field is <d:getcontenttype>text/markdown</d:getcontenttype>

Reply:
yes, lets do that

Reply:
mediaType it is